runners/wikiVsCode/templates/installCobiGen.asciidoc

```diff
@@ -0,0 +1,13 @@
+== Install GobiGen
+Installing devonfw CodeGenerator(i.e. CobiGen).
+
+=== Prerequisites
+CobiGen VSCode extension is installed inside your devonfw distribution. In order to execute it, you have to install the devonfw IDE first. Follow the https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] documentation to install the same.
+
+=== Install CobiGen for VSCode Editor
+
+CobiGen is already installed for your VSCode Editor if you have installed it with the devonfw setup.
```

Comment:
The VS Code plugin is currently not automatically installed within the devon ide. The plugin is available on https://github.com/devonfw-forge/cobigen-vscode-plugin and has to be installed manually. The user can do this via command line or in VS Code via the Command Palette.